js/ngDialog.js

```diff
@@ -5,16 +5,16 @@
  */
 
 (function (root, factory) {
-    if (typeof module !== 'undefined' && module.exports) {
-        // CommonJS
-        module.exports = factory(require('angular'));
-    } else if (typeof define === 'function' && define.amd) {
-        // AMD
-        define(['angular'], factory);
-    } else {
-        // Global Variables
-        factory(root.angular);
-    }
+	if (typeof module !== 'undefined' && module.exports) {
+		// CommonJS
+		module.exports = factory(require('angular'));
+	} else if (typeof define === 'function' && define.amd) {
+		// AMD
+		define(['angular'], factory);
+	} else {
+		// Global Variables
+		factory(root.angular);
+	}
```

Comment:
Its funny but I had them as spaces and changed them because they didn't match the style in the file. I will fix that. I didnt see the contributing file. Ill make sure it matches what was set in that file.